google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/metadata_constants.py

```diff
@@ -0,0 +1,4 @@
+class MetadataDictKeys:
```

Comment:
Class name should follow the convention of the file name: `metadata_constants.py.`

Rename to MetadataConstants ?

Otherwise could rename the file.